week-3/F-array-forEach/exercise.js

```diff
@@ -9,6 +9,20 @@
 
 var arr = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15];
 
+arr.forEach(function(number) {
+  if (number !== 0) {
+    if (number % 5 === 0 && number % 3 === 0) {
+      console.log("FizzBuzz");
+    } else if (number % 3 === 0) {
+      console.log("Fizz");
+    } else if (number % 5 === 0) {
+      console.log("Buzz");
+    } else {
+      console.log(number);
+    }
+  }
+});
```

Comment:
Perfect! I like your check for `0` at the top and then nesting the other checks inside of it :)